dags/openshift_nightlies/config/install/hypershift/sdn.json

```diff
@@ -0,0 +1,18 @@
+{
+    "aws_profile": "",
+    "aws_access_key_id": "",
+    "aws_secret_access_key": "",
+    "rosa_environment": "staging",
+    "rosa_cli_version": "master",
+    "ocm_environment": "stage",
+    "openshift_worker_count": 12,
+    "openshift_network_type": "OpenShiftSDN",
+    "openshift_worker_instance_type": "m5.2xlarge",
+    "machineset_metadata_label_prefix": "machine.openshift.io",
+    "openshift_workload_node_instance_type": "m5.2xlarge",
+    "number_of_hosted_cluster": 2,
+    "hosted_cluster_nodepool_size": 3,
+    "hosted_cluster_instance_type": "m5.2xlarge",
+    "hosted_cluster_network_type": "OpenShiftSDN", 
+    "hosted_control_plane_availability": "SingleReplica" 
```

Comment:
Is HA not working atm? I'd rather default to HA if possible

Reply:
yeah, I can do that,  I am also working on a reasonable `instance_type` for both mgmt as well as hosted nodes.